about.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">    
+    <link rel="stylesheet" href="./scss/aboutstyles.css">
+    <title>Capstone Module 1</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header__social">
+            <ul class="header__social--ul">
+                <li>
+                    <img class="facebook" src="./img/facebook.svg" alt="facebook">
+                    <img src="./img/twitterWhite.svg" alt="Twitter">
+                </li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
```

Comment:
- [ OPTIONAL ] Great job so far @CesarHerr Your project looks great. I will only suggest that just as you adjusted the styling of the top bar to be vertically aligned with the nav bar as the previous reviewer mentioned on the Home page, I will suggest you do the same on the About Page as well :wink: 
- Your Design Now
![image](https://user-images.githubusercontent.com/102699958/233485108-ab7529d3-9ead-41f4-ac9c-ef61f46e0c4e.png)
 - Your Home Page
![image](https://user-images.githubusercontent.com/102699958/233485399-9edb1a4a-c156-4977-9a70-fffd7bf08f05.png)